src/content/ui-api/management-api/authentication/index.md

```diff
@@ -121,13 +121,89 @@ Let's go into details for the most relevant properties (also called "claims") of
 | `name` | Friendly name of the authenticated user. |
 | `preferred_username` | Name to use as an identifier, as an alternative to the user's email address. |
 
-## Authenticating for programmatic access {#service-auth}
```

Comment:
FYI: Changing the anchor will mean that existing links from the outside will no longer find a specific target. So should be avoided if possible.

Reply:
Got it!